daemon/gluster-blockd.c

```diff
@@ -67,8 +75,104 @@ glusterBlockDHelp(void)
 }
 
 
-void *
-glusterBlockCliThreadProc (void *vargp)
+/* [FIXME]
```

Comment:
You should add a reference to a bug report or mailinglist discussion with the libtirpc people. Also file a GitHub issue for yourself as a reminder that this needs to be addressed.

Reply:
@nixpanic do you know tirpc devel mailing list ?
I have debug this and found it myself, at the moment there is not much details that I could find on internet about TIRPC :-(